pinecone/config/config.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 from pinecone.exceptions import PineconeConfigurationError
-from pinecone.core.client.exceptions import ApiKeyError
+from pinecone.core.client.exceptions import PineconeApiKeyError
```

Comment:
Doesn't seem like `ApiKeyError` was used in this file previously. Is this import needed?

Reply:
Iffy. I can add it to the exports list I suppose in case anyone wants to catch that without drilling down into subpackages.